ksmt-test/src/test/kotlin/org/ksmt/test/MultiIndexedArrayTest.kt

```diff
@@ -0,0 +1,445 @@
+package org.ksmt.test
+
+import com.microsoft.z3.Context
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Disabled
+import org.ksmt.KContext
+import org.ksmt.KContext.SimplificationMode.NO_SIMPLIFY
+import org.ksmt.expr.KExpr
+import org.ksmt.expr.rewrite.KExprUninterpretedDeclCollector
+import org.ksmt.solver.KSolver
+import org.ksmt.solver.KSolverStatus
+import org.ksmt.solver.bitwuzla.KBitwuzlaContext
+import org.ksmt.solver.bitwuzla.KBitwuzlaExprConverter
+import org.ksmt.solver.bitwuzla.KBitwuzlaExprInternalizer
+import org.ksmt.solver.bitwuzla.KBitwuzlaSolver
+import org.ksmt.solver.runner.KSolverRunnerManager
+import org.ksmt.solver.z3.KZ3Context
+import org.ksmt.solver.z3.KZ3ExprConverter
+import org.ksmt.solver.z3.KZ3ExprInternalizer
+import org.ksmt.solver.z3.KZ3Solver
+import org.ksmt.sort.KArray2Sort
+import org.ksmt.sort.KArray3Sort
+import org.ksmt.sort.KArraySort
+import org.ksmt.sort.KArraySortBase
+import org.ksmt.sort.KBv8Sort
+import org.ksmt.sort.KBvSort
+import org.ksmt.sort.KSort
+import org.ksmt.utils.uncheckedCast
+import kotlin.test.Test
+import kotlin.test.assertTrue
+import kotlin.time.Duration.Companion.minutes
+
+@Disabled
```

Comment:
Why? Add a comment about it

Reply:
![image](https://user-images.githubusercontent.com/31047452/225901928-f6c047c6-44e3-4ce0-8458-08fc3f73ea1f.png)

Reply:
It's ok. Z3 has some issues with constant arrays equality.